pkgs/applications/networking/cluster/k8sgpt/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "k8sgpt";
+  version = "0.2.9";
```

As new version release allow me to suggest newest version changes.

```suggestion
  version = "0.3.5";
```